api/Controllers/RoomController.cs

```diff
@@ -76,10 +78,11 @@ public async Task<IActionResult> GetRooms([FromRoute] int hotelId)
             {
                 return BadRequest("Such hotel does not exist");
             }
-            Room[] rooms = await _hotelsDb.Rooms
-                .Where(r => r.HotelId == hotelId)
+            List<Room> rooms = await _hotelsDb.Rooms
+                .Where(room => room.HotelId == hotelId)
+                .Include(room => room.Facilities)
                 .ProjectTo<Room>(_mapper.ConfigurationProvider)
-                .ToArrayAsync();
+                .ToListAsync();
```

Comment:
Why?

Reply:
It is inconvenient to map properties sometimes because I might have list in one class, and array in other one, and list implements ICollection, so I can use it as data type for ef navidation properties, I cound not do the same with arrays. Also I often need methods like Add and Remove, and they are able on List but not on array

I would consider using array if to talk about performance in case when I'm sure array methods would be enough

Reply:
I cannot see any changes in logic related to adding/removing functionality. Also I cannot see anything related to EF. That is why I'm wondering why did you change that materialization method